### DISCUSSION TITLE
getting all saved posts from a certain subreddit

### DISCUSSION THREAD

**Discussion by alphonso-a** [Serene-Arc/bulk-downloader-for-reddit#963] [Q&A]
Is it possible to download only saved posts from a certain subreddit? I tried the '--subreddit' and '--saved' tags together but this doesn't seem to work...

**Comment by Serene-Arc** [comment]
No, not directly. The closest that we have is the `--skip-subreddit` option. You can supply that multiple times to remove the subreddits you don't want, otherwise I'd recommend putting the subreddit in the file or folder naming scheme and then just deleting what you don't want.
